net/inadyn/Makefile

```diff
@@ -15,7 +15,7 @@ PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL=https://github.com/troglobit/inadyn
+PKG_SOURCE_URL:=https://github.com/troglobit/inadyn
```

Comment:
Because nothing in my scripts relies on that line being consistent (or even existing).

There are a few other examples of that exact assignment in the feed that I'm assuming have been copy & pasted between Makefiles.